packages/shared/src/components/ProfileMenu.tsx

```diff
@@ -4,10 +4,16 @@ import { Item } from '@dailydotdev/react-contexify';
 import Link from 'next/link';
 import AuthContext from '../contexts/AuthContext';
 import PowerIcon from './icons/Power';
-import { UserIcon } from './icons';
+import { KeyReferralIcon, UserIcon } from './icons';
 import DevCardIcon from './icons/DevCard';
 import SettingsIcon from './icons/Settings';
 import { IconSize } from './Icon';
+import { useLazyModal } from '../hooks/useLazyModal';
+import { LazyModal } from './modals/common/types';
```

Comment:
Don't think you ever need this here.

Reply:
Can you elaborate on why?

Reply:
Why do we need the LazyModal types here, the useLazy will handle everything right?

Reply:
To set the `type` of modal we want to open. For example: `openModal({ type: LazyModal.SearchRefferal })`.